layouts/partials/installation/install/_install_on_containers.md

```diff
@@ -0,0 +1,19 @@
+{{- if eq (.Get "service_name") "" -}}
+{{ errorf "service_name parameter not specified" }}
+{{- end -}}
+
+To run GoCD on container please use our official [docker container images](https://hub.docker.com/u/gocd)
+
+If for some reason want to install gocd using rpm/deb packages on container, from 19.7.0 release additional dependecies needs to be installed before installing GoCD {{ .Get "service_name" }}
```

Comment:
For installing GoCD using rpm/deb packages on containers, starting from release 19.7.0 the following additional dependencies will be required.

Reply:
Done